### PR TITLE
Feature/BSA-201/add table renderer

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/BSA-201

Added renderer for `qtiClass: 'table'`
Added css rules for table.

How to test:
1. install `wkhtmltopdf` https://hub.taotesting.com/techdocs/tao-booklet/readme-booklet
2. set `define('DEBUG_MODE', false);` in `config/generis.conf.php`
3. in `config/taoBooklet/wkhtmltopdf.conf.php` set binary to wkhtmltopdf path for example `'binary' => '/usr/local/bin/wkhtmltopdf',`
3. compile bundles for taoQtiPrint and taoBooklet
`cd tao/views/build`
`npx grunt taoqtiprintbundle` 
`npx grunt taobookletbundle` 
4. create shared stimulus **with table**
5. create items with shared stimulus
6. create test with these items
7. generate booklet from test
8. open booklet in preview and download pdf
9. booklet should render without errors

### CODE REVIEW

File: views/js/qtiPrintRenderer/renderers/Table.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017 (original work) Open Assessment Technologies SA;
```
Comment (dresha-dev) [inline views/js/qtiPrintRenderer/renderers/Table.js]: Everyone wants to skip 2020 🙂
```suggestion
 * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
```

**Review by dresha-dev — APPROVED** [review]

**Review by dresha-dev — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

File: views/js/qtiPrintRenderer/renderers/Table.js (hunk 1 above)
Comment (habemuscode) [reply views/js/qtiPrintRenderer/renderers/Table.js]: Done :)

**Review by habemuscode — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Pull request merged**